docs/docs/workflows/troubleshooting.md

```diff
@@ -35,11 +37,39 @@ Or alternatively, for `terminate`:
 ARM_SKIP_PROVIDER_REGISTRATION=true constellation terminate
 ```
 
-## Cloud logging
+### Nodes fail to join with error `untrusted PCR value`
+
+This error indicates that a node's [attestation statement](../architecture/attestation.md) contains measurements that don't match the trusted values expected by the [JoinService](../architecture/microservices.md#joinservice).
+This may for example happen if the cloud provider updates the VM's firmware such that it influences the [runtime measurements](../architecture/attestation.md#runtime-measurements) in an unforeseen way.
+You can change the expected measurements to resolve the failure.
+
+:::caution
+
+Attestation and trusted measurements are crucial for the security of your cluster.
+Be extra careful when manually changing these settings.
+When in doubt, check if the encountered [issue is known](https://github.com/edgelesssys/constellation/issues?q=is%3Aopen+is%3Aissue+label%3A%22known+issue%22) or [contact support](https://github.com/edgelesssys/constellation#support).
+
+:::
+
+You can use the `upgrade apply` command to change measurements of a running cluster:
+
+1. Modify the `measurements` key in your local `constellation-conf.yaml` to the expected values.
+2. Run `constellation upgrade apply`.
+
+Keep in mind that running `upgrade apply` also applies any version changes you made in your config to the cluster.
+
+You can run these commands to learn about the versions currently configured in the cluster:
+- Kubernetes API server version: `kubectl get nodeversion constellation-version -o json -n kube-system | jq .spec.kubernetesClusterVersion`
+- image version: `kubectl get nodeversion constellation-version -o json -n kube-system | jq .spec.imageVersion`
+- microservices versions: `helm list --filter 'constellation-services' -n kube-system`
+
+## Tools and methods to find the cause of a problem
```

Comment:
```suggestion
##  Diagnosing issues / Debugging tools / Troubleshooting techniques / Debugging / Tooling
```
Trying to make the title more concise.

Reply:
I feel like I have to add an introductory sentence to the section if I choose one of these. Would you prefer that combination over a longer heading? Or do you think it also works without introductory sentence?

Reply:
The first three titles seem descriptive enough to me. For title 4 and 5 I agree. 

Diagnosing issues --> a section about diagnosing issues. 
the section continues to explain how to get logging information and shells. That was my thinking.